conf/default.toml

```diff
@@ -37,6 +37,9 @@ proxy_path = ""
 # Enables displaying a "contact block" wherever relevant in applications
 # contact_email = "opendata@mycompany.com"
 
+# Allows to customize the list of licenses considered as "open data".
```

Comment:
Maybe we should mention the editor here as well?
```suggestion
# Allows to customize the list of licenses inside the metadata editor considered as "open data".
```
WDYT?

Reply:
Yes, good idea :+1: